src/main/pages/che-7/overview/assembly_introduction-to-eclipse-che.adoc

```diff
@@ -34,7 +34,7 @@ factories
 Get started with Eclipse Che by:
 
 * Following the link:{site-baseurl}che-7/running-che-locally[quick-start guide] to get Eclipse Che installed locally on your computer
-* Learning more about Eclipse Che: link:https://www.eclipse.org/che/docs/che-7/introduction-to-eclipse-che/#what-is-eclipse-che[What is Eclipse Che] and link:{site-baseurl}che-7/high-level-che-architecture[Architecture overview]
+* Learning more about Eclipse Che: <<#what-is-eclipse-che>> and link:{site-baseurl}che-7/high-level-che-architecture[Architecture overview]
```

Comment:
this is not correct syntax; see https://asciidoctor.org/docs/asciidoc-syntax-quick-reference/#links

Reply:
To confirm, it should have been <<what-is-eclipse-che>> (no #), correct?